docs/site/todo-tutorial-geocoding-service.md

```diff
@@ -225,6 +225,31 @@ export class TodoController {
 }
 ```
 
+### Extend application with ServiceMixin
```

Comment:
Are you sure we need this step? 

Since #1649, applications scaffolded via `lb4` include the `ServiceMixin` automatically, unless the user chooses to opt-out.

Should we perhaps rephrase this section and explain that it applies only to applications that were initially scaffolded without services?

Reply:
Oh, great catch!! In light of this I think I would like to not see this section added to the docs since we are trying to ready the docs for GA.

@shazha Can you update this to actually just remove the `Installed @loopback/service-proxy` section. The latest version of `@loopback/cli` scaffolds an app correctly and should not require installing / modifying anything else on the part of a user.

Reply:
The `ServiceMixin` is indeed enabled by default. My own application was generated by a earlier version `lb4`, I should've tested it against the latest cli.
I've removed the newly added section, as well as the `Install @loopback/service-proxy` section mentioned by @virkt25 .